packages/snap/snap.manifest.json

```diff
@@ -7,7 +7,7 @@
     "url": "https://github.com/MetaMask/snap-template.git"
   },
   "source": {
-    "shasum": "ZowWdJLCOijpOFAdPXlXnzpqIq2L0pM3m+E/zgCKs/k=",
+    "shasum": "GbpXAntwgt11+Q6D/7MZYNgFRPzbV7lYzmb8DX9ojNo=",
```

Comment:
How come this changed? Is the currently committed shasum not correct?

If not, can we add CI to validate this?

Reply:
Added here: https://github.com/MetaMask/template-snap-monorepo/pull/7